README.md

```diff
@@ -1,2 +1,7 @@
 # hey-look-a-todo-app
-Let's make a ToDo app with @stewartmccoy because he's a real radical dude!
+A Todo app built with Django, Backbone.js, and MongoDB. Developed by [Eric Zarowny](http://www.linkedin.com/in/ezarowny/en) and [Stewart McCoy](http://www.linkedin.com/in/stewartmccoy/en).
+
+To run locally:
+
+1. `workon todo`
```

Comment:
This only works if you have virtualenv installed and named your env 'todo'. Very specific.